state/spacesdiscovery.go

```diff
@@ -121,6 +121,10 @@ func (st *State) SaveSubnetsFromProvider(subnets []network.SubnetInfo, spaceName
 			if modelSubnetIds.Contains(id) {
 				continue
 			}
+			if subnetNet.IP.To4() == nil {
+				logger.Warningf("%s address is not an IPv4 address.", subnetNet.IP)
```

Comment:
I don't think this is a Warning, as users shouldn't submit a bug if they have an IPv6 subnet.
Possibly INFO, but more likely DEBUG.

I think I just missed this line when I was reviewing last time, sorry about that.